fabric.ts

```diff
@@ -178,3 +178,5 @@ export { Control } from './src/controls/Control';
 export * as controlsUtils from './src/controls';
 
 export * from './src/filters';
+
+export { initAligningGuidelines } from './lib/aligning_guidelines/index';
```

Comment:
this is the issue i want to solve. I don't want to package those in this way.
I ll need to work out a way for rollup to produce a different bundle that has fabric has an external dep and that is reacheable from fabric/plugin or something like that